terraform/README.md

```diff
@@ -0,0 +1,23 @@
+###**ECR**
```

Comment:
You need to provide instructions and not links on how to do the things.. :)

Reply:
I've updated the README.md file as per discussed with Kiko.